src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -50,6 +50,8 @@ const IncidentSplitContainer = ({ FormComponent }) => {
     const shouldPatchDirectingDepartment = differentLength || differentValue;
     const shouldPatchNote = Boolean(note?.trim());
 
+    if (!shouldPatchDirectingDepartment && !shouldPatchNote) return null;
```

Comment:
I would rather see you combine this with the `return` statement using a ternary expression.